src/app/variant-reports/variant-reports.service.spec.ts

```diff
@@ -30,13 +32,20 @@ describe('VariantReportsService', () => {
         { provide: APP_BASE_HREF, useValue: '' },
       ]
     });
+
+    service = TestBed.inject(VariantReportsService);
   });
 
-  it('should ...', inject([VariantReportsService], (service: VariantReportsService) => {
+  it('should ...', () => {
     expect(service).toBeTruthy();
-  }));
+  });
+
+
+  it('should get download link', () => {
+    const expectedLink = 'http://localhost:8000/api/v3/common_reports/families_data/undefined';
```

Comment:
I think the expected link should be `'http://localhost:8000/api/v3/common_reports/families_data/test_dataset`, according to the `MockDatasetsService` that is defined at the top of the spec file. Is it possible that you have to configure it further in the `TestBed`?